content/en/docs/concepts/architecture/nodes.md

```diff
@@ -80,53 +80,53 @@ For self-registration, the kubelet is started with the following options:
+  - `--node-labels` - {{< glossary_tooltip text="Labels" term_id="label" >}} to add when registering the node in the cluster (see label restrictions enforced by the [`NodeRestriction` admission plugin](/docs/reference/access-authn-authz/admission-controllers/#noderestriction)).
   - `--node-status-update-frequency` - Specifies how often kubelet posts node status to master.
 
-When the [Node authorization mode](/docs/reference/access-authn-authz/node/) and
-[NodeRestriction admission plugin](/docs/reference/access-authn-authz/admission-controllers/#noderestriction) are enabled,
-kubelets are only authorized to create/modify their own Node resource.
+When the [node authorization mode](/docs/reference/access-authn-authz/node/) and
+[`NodeRestriction` admission plugin](/docs/reference/access-authn-authz/admission-controllers/#noderestriction) are enabled,
+kubelets are only authorized to create/modify their own `Node` resource.
 
-### Manual Node administration
+### Manual node administration
 
-You can create and modify Node objects using
+You can create and modify `Node` objects using
 {{< glossary_tooltip text="kubectl" term_id="kubectl" >}}.
 
-When you want to create Node objects manually, set the kubelet flag `--register-node=false`.
+When you want to create `Node` objects manually, set the kubelet flag `--register-node=false`.
 
-You can modify Node objects regardless of the setting of `--register-node`.
-For example, you can set labels on an existing Node, or mark it unschedulable.
+You can modify `Node` objects regardless of the setting of `--register-node`.
+For example, you can set labels on an existing `Node`, or mark it unschedulable.
 
-You can use labels on Nodes in conjunction with node selectors on Pods to control
-scheduling. For example, you can constrain a Pod to only be eligible to run on
+You can use labels on nodes in conjunction with node selectors on pods to control
+scheduling. For example, you can constrain a pod to only be eligible to run on
 a subset of the available nodes.
 
 Marking a node as unschedulable prevents the scheduler from placing new pods onto
-that Node, but does not affect existing Pods on the Node. This is useful as a
+that node, but does not affect existing pods on the node. This is useful as a
 preparatory step before a node reboot or other maintenance.
 
-To mark a Node unschedulable, run:
+To mark a node unschedulable, run:
 
 ```shell
 kubectl cordon $NODENAME
 ```
 
 {{< note >}}
 Pods that are part of a {{< glossary_tooltip term_id="daemonset" >}} tolerate
-being run on an unschedulable Node. DaemonSets typically provide node-local services
-that should run on the Node even if it is being drained of workload applications.
+being run on an unschedulable node. Daemon sets typically provide node-local services
```

Comment:
? "Daemon sets" == sets of daemons?

Reply:
“DaemonSets” might work better, I reckon.

Reply:
Yes, @bergerhoffer , Would you revert this change to `DaemonSets`?